app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Controllers;
+
+class PostController extends controller
+{
+    public function show($page) {
```

Comment:
Do not forget to handle these generated comments. In the exam there will be a point deduction when these are found in your code.